src/components/image/Image.tsx

```diff
@@ -6,24 +6,13 @@ type Image = {
   width: number;
   height: number;
   altText: string;
-  caption: string;
-  attribution: string;
   alignment: "left" | "center" | "right";
 };
 
 // const [isHovering, setIsHovering] = useState(false)
 
-const Images: FC<Image> = ({
-  image,
-  width,
-  height,
-  altText,
-  caption,
-  attribution,
-  alignment,
-}) => {
+const Images: FC<Image> = ({ image, width, height, altText, alignment }) => {
```

Comment:
**issue (blocking):** Other pages rely on properties like `caption`.  Making these optional is being addressed in #220.  In the meantime can you just use the NextJS <Image> component directly?  Alternative we can wait for that issue to be resolved and then you can use the updated version of this component.